Populist Legal Power/common/political_movements/yMog_PLP_religious_movements.txt

```diff
@@ -224,6 +224,10 @@
 		weight = {
 			value = 100
 		}
+		
+		interest_group_can_join = {
```

Comment:
Interesting, is this a new block that's just been added, or was it just not part of these movement definitions before?